src/test/java/org/terasology/namegenerator/CreatureNameGeneratorSystemTest.java

```diff
@@ -22,7 +22,7 @@
  * Tests {@link CreatureNameGeneratorSystem}
  */
 @ExtendWith(MTEExtension.class)
-@Dependencies("NameGenerator")
+@IntegrationEnvironment(dependencies = "NameGenerator")
 @Tag("MteTest")
```

Comment:
Same comment as for the other module. With the new `IntegrationEnvironment` annotation, the `ExtendWith` and `Tag` annotations are not longer required, so please remove them as well. This will also count for any future modules you refactor in this context :wink: